readme.md

```diff
@@ -21,8 +22,8 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://vanvalera.github.io/layout_stars/)
+- [TEST REPORT LINK](https://vanvalera.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="946" alt="image" src="https://github.com/user-attachments/assets/06e38512-2d64-4f0e-bdb6-318cce43e8e3">